src/main/java/edu/rosehulman/covidtracer/model/UserRole.java

```diff
@@ -0,0 +1,31 @@
+package edu.rosehulman.covidtracer.model;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+public class UserRole implements Serializable {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	@Column(name = "ID", nullable = false)
+	private Long ID;
+
+	@Column(name = "Role", nullable = false, unique = true)
+	private ROLES ROLES;
```

Comment:
I would probably advise against a private field with the same name as your ENUM. I can follow your code below, but its not readily readable. Also this should be singularized.

Reply:
Retweet